blenderproc/python/constructor/RandomRoomConstructor.py

```diff
@@ -124,6 +124,8 @@ def construct_random_room(used_floor_area: float, interior_objects: List[MeshObj
         if is_duplicated:
             # delete the duplicated object
             list_of_deleted_objects.append(current_obj)
+            for child in current_obj.get_children():
```

Comment:
But what if there are more than two levels and the children have again child objects?
Wouldn't it be more convenient to add a `recursive` parameter to the `deleteMultiple` functions which would collect and delete all child/grandchild etc. objects?

Reply:
Absolutely :D 

I just wanted to see if you review my code correctly ;) 

I totally agree. The parameter should be added to the delete function.